golos.vesting/golos.vesting.hpp

```diff
@@ -34,10 +34,21 @@ class vesting : public contract {
         const auto& balance = balances.get(sym.raw());
         return balance.vesting;
     };
-    inline asset get_account_effective_vesting(name account, symbol_code sym) const;
+    static inline asset get_account_effective_vesting(name code, name account, symbol_code sym) {
```

Comment:
Why do you have to change the signature of these methods? 
And why don't you change `get_account_available_vesting`?
IMHO: Changing these methods is not part of this task. For a token contract, this was done because compatibility with the original code is required there.